packages/cart/src/services/cart-module.ts

```diff
@@ -90,10 +90,12 @@ export default class CartModuleService implements ICartModuleService {
 
   @InjectManager("baseRepository_")
   async create(
-    data: CreateCartDTO[],
+    data: CreateCartDTO[] | CreateCartDTO,
```

Comment:
I think that we are missing the overload in this service as well for the methods. Also, the return should be updated to a single or multiple object 😄